lib/active_merchant/billing/gateways/commerce_hub.rb

```diff
@@ -2,7 +2,7 @@ module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class CommerceHubGateway < Gateway
       self.test_url = 'https://connect-cert.fiservapps.com/ch'
-      self.live_url = 'https://prod.api.fiservapps.com/ch'
+      self.live_url = 'https://connect.fiservapis.com/ch/'
```

Comment:
Two questions 

1. Is the `/` that was added to the end going to be an issue? 
2. How confident are we that this url is going to be good to go? I just worry since our remote tests all use the test_url.

Reply:
I've removed the '/' it was in the ticket but it doesn't need to be added

Reply:
Perfect!